frontend/src/components/MapLibreMap.vue

```diff
@@ -64,27 +73,32 @@ let draw: MapboxDraw | undefined = undefined;
 
 onMounted(() => {
   map = new Map({
-    container: "maplibre-map",
-    center: [props.center[1], props.center[0]],
+    container: mapContainer.value as HTMLElement,
+    center: props.bounds ? undefined : [props.center[1], props.center[0]],
+    bounds: props.bounds,
     style: props.styleSpec,
     trackResize: true,
     zoom: props.zoom,
   });
+
   map.addControl(new NavigationControl({}));
-  map.addControl(new GeolocateControl({}));
   map.addControl(new ScaleControl({}));
-  map.addControl(new FullscreenControl({}));
   draw = new MapboxDraw({
     displayControlsDefault: false,
   });
-  map.addControl(draw as unknown as IControl);
-  map.addControl(
-    new MaplibreGeocoder(geocoderApi, {
-      maplibregl: { Marker },
-      showResultsWhileTyping: true,
-    }),
-    "top-left"
-  );
+
+  if (props.extended) {
+    map.addControl(new GeolocateControl({}));
+    map.addControl(new FullscreenControl({}));
+    map.addControl(draw as unknown as IControl);
+    map.addControl(
+      new MaplibreGeocoder(geocoderApi, {
+        maplibregl: { Marker },
+        showResultsWhileTyping: true,
```

Comment:
You could specify the current locale, see example in 
https://github.com/EPFL-ENAC/alice-perl-eerl-urbtrees/blob/main/src/components/MapLibreMap.vue#L98